index.html

```diff
@@ -41,48 +43,51 @@ <h2><a class = "logo" href = "#">Ki Vera Luna Pardillo</a></h2>
 
 
             </header> <br> <br> <br>
-            <section id = "about">
+
+            
+            <section class="section_padding" id = "about">
                 <h1>About</h1>
-                <p>My name is Ki Vera Luna. I am from the Philippines. My major is Computer Information System. </p>
+                <p class = "intro">My name is Ki Vera Luna. I am from the Philippines. My major is Computer Information System. </p>
             </section>
 
-            <section class="list_experience" id = "experience">
-            <div class = "flexing">
-                <h1 class = "exp_h1">Experience </h1>
+            
+
+            <section class="section_padding" id = "experience">
+                <h1>Experience </h1> 
+
                 <ul class = "flexs">
                     <li class = "flex-box one">Gateway Technology Center Receptionist</li>
                     <li class = "flex-box two">Geodetic Survey Aplication Developer Summer Intern</li>
                     <li class = "flex-box three">Code the Dream Student </li>
+                    <li class = "flex-box four">Computer Information System Student  </li>
                 </ul>
-            </div>
-            </section><br><br><br><br>
-
-            <div class="flex-skills">
-                <section id = "skills" >
-                    <h1 class = "skill_h2"> Skills</h1>
-                    <ul class = "skills_ul">
-                        
-                    </ul>
-                
+            
+            </section>
+            
 
-                </section>
-            </div> <br><br><br>
+            
+            <section class="section_padding" id = "skills" >
+                <h1 class = "skill_h2"> Skills</h1>
+                <ul class = "skills_ul">            
+                </ul>
+            </section>
+            
 
-            <section id ="projects">
+            <section class="section_padding" id ="projects">
                 <h1>Projects</h1>
-                <ul>
+                <ul class = "projects_ul">
 
 
                 </ul>
             </section>
 
-            <section id = "connect">
+            <section class="section_padding" id = "connect">
                 <h1>Connect </h1>
-                <a href= https://www.linkedin.com/in/kipardillo/> linkedin </a>
+                <a class="connect_a" href= https://www.linkedin.com/in/kipardillo  target="_blank"><i class="fab fa-linkedin"></i>  Linkedin </a>
             </section>
 
-            <section id = "contact">
-                <h2> Leave a Message</h2>
+            <section class="section_padding" id = "contact">
+                <h1> Leave a Message</h1>
                 <form name = "leave_message">
                 
                     <label for  = leave_message> </label>
```

Comment:
I think this particular label is not being used anywhere.

Great job adding the labels for the other input controls; the only thing missing is that each form control (`input` or `textarea` etc) should have an `id` attribute that matches the associated label's `for` attribute